### DISCUSSION TITLE
Github education

### DISCUSSION THREAD

**Discussion by va1ker** [community/community#137726] [GitHub Education]
### Hi everyone,

Im provide all info about myself, even my university id email, email for teachers, and other stuff, and my account was not approved two times. What should i do? Im already filled billing info form, but even with it, im not approved...

**Comment by md-ashraful-islam-ullas** [comment]
Hi there,

When your request is rejected, GitHub usually sends a message through your registered email explaining what’s missing or wrong. You should check that message to see what needs to be fixed or updated in your application.

Sometimes there might be an issue with the format or the way the documents were submitted. For more information, Check this #111352

**Reply by va1ker** [reply]
They sending an automatic message with reject with general reasons. No anything specific, i send ALL info i can. If i could, i would send more info then now(they accept only 1 photo-proof). But they still reject me, so i need to contact with someone i guess

**Comment by Injamul2k24** [comment]
Check you mail  .  there will maybe some information 

Join this committee and and your problem : https://github.com/orgs/Hoqueinja/discussions/1
